.github/workflows/nix.yml

```
@@ -31,4 +31,4 @@ jobs:
           ./result/bin/amber --version
 
           # Ensure that the `bc` command is correctly provided
-          echo "import { exit } from \"std/env\"\n\$which bc && bc --version\$ failed { echo \"Failed to run basic calculator\"\nexit(1) }" | ./result/bin/amber -
+          echo "import { exit } from \"std/env\" unsafe \$which bc && bc --version\$ failed { echo \"Failed to run basic calculator\" exit(1) }" | ./result/bin/amber -
```

The command is already handled within the failed block so there is no need for an unsafe keyword.


```suggestion
          echo "import { exit } from \"std/env\" \$which bc && bc --version\$ failed { echo \"Failed to run basic calculator\" exit(1) }" | ./result/bin/amber -
```